pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandlerTest.java

```diff
@@ -1204,9 +1204,15 @@ public void testEnableDictionaryForMultipleColumns()
     IndexLoadingConfig indexLoadingConfig = new IndexLoadingConfig(null, _tableConfig);
 
     Random rand = new Random();
-    String col1 = _noDictionaryColumns.get(rand.nextInt(_noDictionaryColumns.size()));
+    String col1;
+    do {
+      col1 = _noDictionaryColumns.get(rand.nextInt(_noDictionaryColumns.size()));
+    } while (FORWARD_INDEX_DISABLED_RAW_COLUMNS.contains(col1));
     indexLoadingConfig.getNoDictionaryColumns().remove(col1);
-    String col2 = _noDictionaryColumns.get(rand.nextInt(_noDictionaryColumns.size()));
+    String col2;
```

Comment:
Should we also ensure `col2` is different from `col1`

Reply:
good catch, done